commit dd53f6102c30a774e0db8e55d49017a38060f6f6
Merge: 59c5c58c5b93 9eecfc22e0bf
Author: Paolo Bonzini <pbonzini@redhat.com>
Date:   Wed May 15 23:41:43 2019 +0200

    Merge tag 'kvmarm-for-v5.2' of git://git.kernel.org/pub/scm/linux/kernel/git/kvmarm/kvmarm into HEAD
    
    KVM/arm updates for 5.2
    
    - guest SVE support
    - guest Pointer Authentication support
    - Better discrimination of perf counters between host and guests
    
    Conflicts:
            include/uapi/linux/kvm.h

diff --cc include/uapi/linux/kvm.h
index d727adf07801,4dc34f8e29f6..2fe12b40d503
--- a/include/uapi/linux/kvm.h
+++ b/include/uapi/linux/kvm.h
@@@ -986,10 -986,11 +986,13 @@@ struct kvm_ppc_resize_hpt 
  #define KVM_CAP_HYPERV_ENLIGHTENED_VMCS 163
  #define KVM_CAP_EXCEPTION_PAYLOAD 164
  #define KVM_CAP_ARM_VM_IPA_SIZE 165
 -#define KVM_CAP_MANUAL_DIRTY_LOG_PROTECT 166
 +#define KVM_CAP_MANUAL_DIRTY_LOG_PROTECT 166 /* Obsolete */
  #define KVM_CAP_HYPERV_CPUID 167
 -#define KVM_CAP_ARM_SVE 168
 -#define KVM_CAP_ARM_PTRAUTH_ADDRESS 169
 -#define KVM_CAP_ARM_PTRAUTH_GENERIC 170
 +#define KVM_CAP_MANUAL_DIRTY_LOG_PROTECT2 168
 +#define KVM_CAP_PPC_IRQ_XIVE 169
++#define KVM_CAP_ARM_SVE 170
++#define KVM_CAP_ARM_PTRAUTH_ADDRESS 171
++#define KVM_CAP_ARM_PTRAUTH_GENERIC 172
  
  #ifdef KVM_CAP_IRQ_ROUTING